full/Dockerfile

```diff
@@ -131,10 +79,10 @@ ENV GOPATH=$HOME/go-packages
 ENV GOROOT=$HOME/go
 ENV PATH=$GOROOT/bin:$GOPATH/bin:$PATH
 RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar xzs && \
-# install VS Code Go tools from https://github.com/Microsoft/vscode-go/blob/0faec7e5a8a69d71093f08e035d33beb3ded8626/src/goInstallTools.ts#L19-L45
+# install VS Code Go tools from https://github.com/golang/vscode-go/blob/74e266bf623fd64b5b0c913db5bbe62873c46bbd/src/goTools.ts
     go get -u -v \
         github.com/mdempsky/gocode \
-        github.com/uudashr/gopkgs/cmd/gopkgs \
+        github.com/uudashr/gopkgs/cmd/gopkgs@v2 \
```

Comment:
Please rebase on top of latest `master` (this change should automatically disappear)